LocalOrbit/localorbit
# Stripe webhook upgrades lo 1045

Update stripe_event gem to support signed webhooks requests from Stripe.

Deployment:
- [ ] configure `STRIPE_SIGNING_SECRET` env var on `staging` and `production`

To test on dev or staging:
- create a new plan (product) at Stripe and verify it appears in the plans database table

TODO:

- [ ] figure out how to invoke an invoice.payment_succeeded event
- [ ] figure out how to invoke an invoice.payment_failed event
